calculator.py

```diff
@@ -108,10 +123,14 @@ def divide(self,a: float,b: float) -> float:
 
 
 def start_calculator():
+    t = threading.Thread(target=thread_additions,)
+    t.start()
+    cal= Calculator()
     args = create_parser()
     salusion=take_expresion(args.f)
-    cal= Calculator()
     print(cal.calculate(salusion))
+    print(f" Total operations: {cal.add.count_operation + cal.subtract.count_operation + cal.multiply.count_operation + cal.divide.count_operation} \n Add operations: {cal.add.count_operation} \n Subtract operations: {cal.subtract.count_operation} \n Multiply operations: {cal.multiply.count_operation} \n Divide operations: {cal.divide.count_operation} \n") # pylint: disable=no-member
```

Comment:
You can do this in a cleaner way. You can use multiline strings (`'''this'''`) or at least split the string in some points to make the line shorter.